scenarios/command/README.md

```diff
@@ -4,14 +4,28 @@
 
 This scenario simulates the request-response messaging pattern. Request-response uses two topics, one for the request and one for the response.
 
-Consider a use case where a user can unlock their car from a mobile app. The request to unlock is published on `vehicles/<vehicleId>/command/unlock/request` and the response of unlock operation is published on `vehicles/<vehicleId>/command/unlock/response`.
+Consider a use case where a user can unlock their car from a mobile app. The request to unlock is published on `vehicles/<vehicleId>/commands/unlock/request` and the response of unlock operation is published on `vehicles/<vehicleId>/commands/unlock/response`. 
+
+## Command Server, Command Client
+
+Every command requires a `server` who implements the command and a `client` who invokes the command, in this case the vehichle is the server and the mobile-app is the client.
 
 |Client|Role|Operation|Topic/Topic Filter|
 |------|----|---------|------------------|
-|vehicle03|server|sub|vehicles/vehicle03/command/unlock/request|
-|vehicle03|server|pub|vehicles/vehicle03/command/unlock/response|
-|mobile-app|client|pub|vehicles/vehicle03/command/unlock/request|
-|mobile-app|client|sub|vehicles/vehicle03/command/unlock/response|
+|vehicle03|server|sub|vehicles/vehicle03/commands/unlock/request|
+|vehicle03|server|pub|vehicles/vehicle03/commands/unlock/response|
+|mobile-app|client|pub|vehicles/vehicle03/commands/unlock/request|
+|mobile-app|client|sub|vehicles/vehicle93/commands/unlock/response|
+
+## Command flow with user properties
+
+To implement the command pattern, the mqtt message used for the request includes additional metadata to control the command flow:
+
+- `Correlation Id` The client includes a new _Guid_ in the message property _CorrelationData_.
```

Comment:
```suggestion
- `Correlation Id` The server includes a new _Guid_ in the message property _CorrelationData_.
```

Reply:
is the client who sets the CorrelationId

Reply:
ah whoops - got things mixed up in my head